playbooks/byo/openshift-cluster/upgrades/v3_3/upgrade.yml

```diff
@@ -58,10 +58,10 @@
 - include: ../../../../common/openshift-cluster/upgrades/pre.yml
   vars:
     openshift_deployment_type: "{{ deployment_type }}"
-- include: ../../../../common/openshift-cluster/upgrades/upgrade.yml
-  vars:
-    openshift_deployment_type: "{{ deployment_type }}"
-    master_config_hook: "v3_3/master_config_upgrade.yml"
-    node_config_hook: "v3_3/node_config_upgrade.yml"
-- include: ../../../openshift-master/restart.yml
+#- include: ../../../../common/openshift-cluster/upgrades/upgrade.yml
+  #vars:
+    #openshift_deployment_type: "{{ deployment_type }}"
+    #master_config_hook: "v3_3/master_config_upgrade.yml"
+    #node_config_hook: "v3_3/node_config_upgrade.yml"
+#- include: ../../../openshift-master/restart.yml
```

Comment:
Something very serious and definitely not intentional slipped into the PR. My bad, fixing now.